### PR TITLE
Content updates for the 2024 year

### PR DESCRIPTION
**Pull request by TobiasGrether**
Targets changes from #660

### CODE REVIEW

File: database/seeders/EventsErstiwocheSeeder.php [hunk 1]
```diff
@@ -345,14 +338,14 @@ public function runSport(): void
         $event->description = '<p>Auch sportliche Aktivitäten kommen bei uns nicht zu kurz. Für eine Anmeldegebühr von <strong>5€</strong> könnt ihr euch am Freitag auspowern.</p>
             <p>Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben.</p>
             <p>Bitte beachtet auch die folgenden Hinweise zu den einzelnen Programmpunkten:</p>
-            <p><strong>Fußball, Volleyball, Basketball:</strong> Die Anmeldegebühr bekommt ihr zurückerstattet, wenn es wegen schlechtem Wetter ausfällt.</p>
+            <p><strong>Hochschulsport:</strong>Bitte bringt 5€ an Pfand mit</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:341]: Please use `Fußball, Volleyball, Basketball` as name and not `Hochschulsport`, this is easier to understand for our first semester students.

File: database/seeders/EventsErstiwocheSeeder.php [hunk 2]
```diff
@@ -363,25 +356,24 @@ public function runSport(): void
         // create event slots
         $slots = [
             [
-                'name' => 'Fußball, Volleyball, Basketball',
+                'name' => 'Hochschulsport',
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:359]: Please use `Fußball, Volleyball, Basketball` as name and not `Hochschulsport`, this is easier to understand for our first semester students.

**Review by simonostendorf — CHANGES_REQUESTED** [review]
Please adjust the comments for the "Katerbrunch", "Sport" and "Kultur" slots.

File: database/seeders/EventsErstiwocheSeeder.php [hunk 3]
```diff
@@ -248,11 +239,19 @@ public function runHausfuehrung(): void
...
                 'course_ids' => [
                     $coursesByAbbreviation['WI']->id,
-                    $coursesByAbbreviation['IS-Master']->id
                 ],
             ];
         }
 
+        for ($i = 1; $i <= 1; $i++) {
+          $groups[] = [
+            'name' => "IS Master Hausführung $i",
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:248]:
```suggestion
            'name' => "IS-Master Hausführung $i",
```

File: database/seeders/EventsErstiwocheSeeder.php [hunk 4]
```diff
@@ -329,11 +330,11 @@ public function runKaterbrunch(): void
         $event->name = 'Katerbrunch';
         $event->description = '<p>Nachdem wir alle nach der Kneipentour am Mittwoch Abend etwas verkatert sind, gibt es doch nichts besseres als zusammen bei einem guten Fr&uuml;hst&uuml;ck auszukatern 😊 <br />Hierf&uuml;r bitte wir euch die 2&euro; Anmeldegeb&uuml;hr am Montag zwischen 14:15 und 15:00 Uhr oder Mittwoch zwischen 09:30 und 13:00 Uhr im FSR zu bezahlen, sonst k&ouml;nnt ihr leider nicht teilnehmen.</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:331]: The time slot where the money can be paied is not correct. Currently I am waiting for an answer for the correct time slot.

File: database/seeders/EventsErstiwocheSeeder.php [hunk 5]
```diff
@@ -329,11 +330,11 @@ public function runKaterbrunch(): void
...
         <p><strong>Wann:</strong> 28.09 ab 12:30 Uhr <br /><strong>Wo:</strong> FH, am D Geb&auml;ude <br /><strong>Was mitbringen:</strong> Tasse/ Becher und Teller ggf, Picknickdecke bei gutem Wetter.</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:332]:
```suggestion
        <p><strong>Wann:</strong> 28.09 ab 12:00 Uhr <br /><strong>Wo:</strong> FH, am D Geb&auml;ude <br /><strong>Was mitbringen:</strong> Tasse/ Becher und Teller ggf, Picknickdecke bei gutem Wetter.</p>
```

12:00 was printed on the time table that was send to the first semester telegram channel.

File: database/seeders/EventsErstiwocheSeeder.php [hunk 6]
```diff
@@ -367,14 +368,15 @@ public function runSport(): void
         $event->description = '<p>Auch sportliche Aktivitäten kommen bei uns nicht zu kurz. Für eine Anmeldegebühr von <strong>5€</strong> könnt ihr euch am Freitag auspowern.</p>
             <p>Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben.</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:369]: Please also add the time slot for payments here (see comments above)

File: database/seeders/EventsErstiwocheSeeder.php [hunk 7]
```diff
@@ -367,14 +368,15 @@ public function runSport(): void
...
             <p>Bitte beachtet auch die folgenden Hinweise zu den einzelnen Programmpunkten:</p>
-            <p><strong>Fußball, Volleyball, Basketball:</strong> Die Anmeldegebühr bekommt ihr zurückerstattet, wenn es wegen schlechtem Wetter ausfällt.</p>
+            <p><strong>Fu0ball:</strong>Bitte bringt 5€ an Pfand mit</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:371]:
```suggestion
            <p><strong>Fußball, Volleyball, Basketball:</strong>Die Anmeldegebühr fungiert als Pfand, welches ihr beim Erscheinen wieder zurückbekommt.</p>
```

File: database/seeders/EventsErstiwocheSeeder.php [hunk 8]
```diff
@@ -436,14 +436,14 @@ public function runKultur(): void
...
             Auch das ist am Freitag in der Erstiwoche möglich.</p>
             <p>Bitte beachtet die folgenden Hinweise zu den einzelnen Programmpunkten:</p>
-            <p><strong>Stadtführung, Tierpark, Tour durch das Alemaniastadion:</strong> Für eine Anmeldegebühr von <strong>5€</strong> ist eine Teilnahme an diesen Angeboten möglich. Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben. Die Anmeldegebühr fungiert als Pfand, welches ihr beim Erscheinen der Kulturangebote wieder zurückbekommt.</p>
-            <p><strong>Kebabtour:</strong> Ihr müsst eure Döner/ Falafel-Taschen selber zahlen.</p>
+            <p><strong>Tierpark, Tour durch das Alemaniastadion:</strong> Für eine Anmeldegebühr von <strong>5€</strong> ist eine Teilnahme an diesen Angeboten möglich. Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben. Die Anmeldegebühr fungiert als Pfand, welches ihr beim Erscheinen der Kulturangebote wieder zurückbekommt.</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:439]:
```suggestion
            <p><strong>Tierpark:</strong> Für eine Anmeldegebühr von <strong>5€</strong> ist eine Teilnahme möglich. Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben. Die Anmeldegebühr fungiert als Pfand, welches ihr beim Erscheinen wieder zurückbekommt.</p>
```

Please also add the time slot for payments here.

File: database/seeders/EventsErstiwocheSeeder.php [hunk 9]
```diff
@@ -436,14 +436,14 @@ public function runKultur(): void
...
+            <p><strong>Foodtour:</strong> Ihr müsst eure Döner / Falafel-Taschen selber zahlen.</p>
             <p><strong>Allgemein:</strong> Anschließende Teilnahme an weiteren Programmpunkten ist nur die “Kebabtour” zeitlich möglich.</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:441]:
```suggestion
            <p><strong>Allgemein:</strong> Eine anschließende Teilnahme an weiteren Programmpunkten ist nur mit der “Foodtour” zeitlich möglich.</p>
```

File: database/seeders/EventsErstiwocheSeeder.php [hunk 10]
```diff
@@ -367,14 +368,15 @@ public function runSport(): void
         $event->description = '<p>Auch sportliche Aktivitäten kommen bei uns nicht zu kurz. Für eine Anmeldegebühr von <strong>5€</strong> könnt ihr euch am Freitag auspowern.</p>
             <p>Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben.</p>
             <p>Bitte beachtet auch die folgenden Hinweise zu den einzelnen Programmpunkten:</p>
-            <p><strong>Fußball, Volleyball, Basketball:</strong> Die Anmeldegebühr bekommt ihr zurückerstattet, wenn es wegen schlechtem Wetter ausfällt.</p>
+            <p><strong>Fu0ball:</strong>Bitte bringt 5€ an Pfand mit</p>
             <p><strong>Yoga:</strong> Bitte bringt eine eigene Yogamatte mit.</p>
+            <p><strong>Bouldern;</strong> Falls nicht vorhanden, können Stoppersocken vor Ort für 3€ erworben werden.</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:373]:
```suggestion
            <p><strong>Bouldern:</strong> Falls nicht vorhanden, können Stoppersocken vor Ort für 3€ erworben werden.</p>
```

File: database/seeders/EventsErstiwocheSeeder.php [hunk 11]
```diff
@@ -367,14 +368,15 @@ public function runSport(): void
...
             <p><strong>Allgemein:</strong> Anschließende Teilnahme an den Kulturprogrammpunkten ist nur die “Kebabtour” zeitlich möglich.</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:374]:
```suggestion
            <p><strong>Allgemein:</strong> Eine anschließende Teilnahme an weiteren Programmpunkten ist nur mit der “Foodtour” zeitlich möglich.</p>
```

**Review by simonostendorf — CHANGES_REQUESTED** [review]

File: database/seeders/EventsErstiwocheSeeder.php (hunk 11 above)
Comment (simonostendorf) [reply database/seeders/EventsErstiwocheSeeder.php:374]: This must be validated, there are new time slots for each event. Please see the "schichtplan".

File: database/seeders/EventsErstiwocheSeeder.php (hunk 9 above)
Comment (simonostendorf) [reply database/seeders/EventsErstiwocheSeeder.php:441]: This must be validated, there are new time slots for each event. Please see the "schichtplan".

File: database/seeders/EventsErstiwocheSeeder.php (hunk 4 above)
Comment (simonostendorf) [reply database/seeders/EventsErstiwocheSeeder.php:331]: 13:45 - 16:00

File: database/seeders/EventsErstiwocheSeeder.php [hunk 12]
```diff
@@ -385,25 +387,24 @@ public function runSport(): void
         // create event slots
         $slots = [
             [
-                'name' => 'Fußball, Volleyball, Basketball',
+                'name' => 'Fu0ball, Volleyball',
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:390]:
```suggestion
                'name' => 'Fußball, Volleyball',
```

File: database/seeders/EventsErstiwocheSeeder.php (hunk 9 above)
Comment (simonostendorf) [reply database/seeders/EventsErstiwocheSeeder.php:441]: time slots:

| slot                        | time               |
|---------------------|----------------|
| Fußball, Volleyball | 12:00 - 15:00 |
| Bouldern               | 13:00 - 16:00 |
| Yoga                      | 12:00 - 13:30 |
| Lasertag                | 12:20 - 14:40 |
| Foodtour               | 15:30 - 17:00 |
| Tierpark                 | 15:30 - 17:00 |

File: database/seeders/EventsErstiwocheSeeder.php (hunk 11 above)
Comment (simonostendorf) [reply database/seeders/EventsErstiwocheSeeder.php:374]: time slots:

| slot                        | time               |
|---------------------|----------------|
| Fußball, Volleyball | 12:00 - 15:00 |
| Bouldern               | 13:00 - 16:00 |
| Yoga                      | 12:00 - 13:30 |
| Lasertag                | 12:20 - 14:40 |
| Foodtour               | 15:30 - 17:00 |
| Tierpark                 | 15:30 - 17:00 |

File: database/seeders/EventsErstiwocheSeeder.php (hunk 4 above)
Comment (TobiasGrether) [reply database/seeders/EventsErstiwocheSeeder.php:426]: > 13:45 - 16:00

For monday and wednesday?

Comment (simonostendorf) [reply database/seeders/EventsErstiwocheSeeder.php:426]: > For monday and wednesday?

sorry, these are the current time slots:

Montag: 13:45 - 16:00
Mittwoch: 12:00 - 15:00

File: database/seeders/EventsErstiwocheSeeder.php [hunk 13]
```diff
@@ -460,16 +461,17 @@ public function runSport(): void
         $event = new Event;
         $event->name = 'Sport';
         $event->description = '<p>Auch sportliche Aktivitäten kommen bei uns nicht zu kurz. Für eine Anmeldegebühr von <strong>5€</strong> könnt ihr euch am Freitag auspowern.</p>
-            <p>Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben.</p>
+            <p>Bitte bezahlt die Anmeldegebühr am Montag zwischen 13:45 und 16:00 Uhr oder Mittwoch zwischen 12:00 und 14:30 Uhr im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben.</p>
             <p>Bitte beachtet auch die folgenden Hinweise zu den einzelnen Programmpunkten:</p>
-            <p><strong>Fußball, Volleyball, Basketball:</strong> Die Anmeldegebühr bekommt ihr zurückerstattet, wenn es wegen schlechtem Wetter ausfällt.</p>
+            <p><strong>Fußball, Volleyball:</strong>Die Anmeldegebühr fungiert als Pfand, welches ihr beim Erscheinen wieder zurückbekommt.</p>
```
Comment (simonostendorf) [inline database/seeders/EventsErstiwocheSeeder.php:466]:
```suggestion
            <p><strong>Fußball, Volleyball:</strong> Die Anmeldegebühr fungiert als Pfand, welches ihr beim Erscheinen wieder zurückbekommt.</p>
```

**Review by simonostendorf — CHANGES_REQUESTED** [review]

**Review by simonostendorf — APPROVED** [review]

**Pull request merged**